arroyo/backends/local/storages/file.py

```diff
@@ -27,6 +28,15 @@
 from arroyo.utils.codecs import Codec
 
 
+@dataclass(unsafe_hash=True)
```

Comment:
I'm just curious, why `unsafe_hash` is used for these classes?

Reply:
I copied this over but I think it's needed otherwise the object won't be hashable.